contrib/polybar.xbps/template

```diff
@@ -1,17 +1,16 @@
 # Template file for 'polybar'
 pkgname=polybar
-version=3.0.4
+version=3.1.0
 revision=1
-_i3ipcpp_version=0.6.2
-_xpp_version=1.3.6
+_i3ipcpp_version=0.7.0
```

Comment:
The 0.7.0 tag on i3ipcpp does not contain the fixes for the jsoncpp and the gcc with jsoncpp < 1.7.7 issues. I will shortly push the 0.7.1 tag so we can fix that

Reply:
I just pointed to the versions that's specified in the `3.1.0` release. This is the template that is merged into the official void repository.

Reply:
Where did we specify a i3ipcpp version for the 3.1.0 release? I don't remember doing that.

I have now pushed the v0.7.1 tag pointing to the newest commit.

Reply:
The submodule points to the `v0.7.0` tag in the i3ipcpp repository.

`git checkout 3.1.0 && cat .gitmodules`